### PR TITLE
importmap command

### LINKED ISSUE #155: importmap command
add a `importmap` command to manage the `import_map.json`:

```
$ aleph importmap add esm.sh/react

  -> v17.0.1
  -  v16.14.0
  -  v16.13.1
  -  other

$ aleph importmap add deno.land/x/aleph@1.0.0
$ aleph importmap update
```

### PR DESCRIPTION
**Pull request by argonaut0**
Closes #155

### CODE REVIEW

File: cli/importmap.ts [hunk 1]
```diff
@@ -0,0 +1,512 @@
+import { exists } from 'https://deno.land/std@0.96.0/fs/mod.ts'
```
Comment (ije) [inline cli/importmap.ts:1]: pls align std to 0.106

File: cli/importmap.ts [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+    -r, --remove <module>            Remove a module from the import map.
+____________________________________________________________________________
+    -u, --update                     Update all modules in the import map.
+
+        --update <module>            Update a module to the latest version.
+____________________________________________________________________________
+    -h, --help                        Prints this help message
+____________________________________________________________________________
+Examples:
+ - moduleID -> address
```
Comment (ije) [inline cli/importmap.ts:33]: can we add some npm examples?

**Comment by ije** [comment]
@kittywantsbacon thanks, this looks great! i have some suggestions for this PR please consider:
- rename `aleph importmap` to `aleph use`
- `aleph use deno.land/std` -> select version -> select path -> auto alias , for example: `aleph use deno.land/std` [@0.106.0] / [path] / [mod.ts] -> `"path": "https://deno.land/std@0.106.0/path/mod.ts"`
- `aleph use deno.land/x` -> select module -> select version -> select path -> auto alias 
- `aleph use esm.sh/react` -> select version -> `"react": "https://esm.sh/react@VER"`
- `aleph use esm.sh/react@17`-> `"react": "https://esm.sh/react@17.0.2"`

**Comment by RaitoBezarius** [comment]
Hi there, I'm super interested into this feature, can I help to get it merged?

**Comment by ije** [comment]
i think we should not add this command, this should be handle by other tools, vscode extension for example or a standalone command tool, thanks a lot @argonaut0

**Pull request closed without merging**